env_stat_utility/cgroup_stats.py

```diff
@@ -83,3 +83,19 @@ def get_io_pressure(self):
         except Exception as e:
             print(f"Error reading IO pressure: {e}")
             return None
+
+    # Пока захардкодил рандомный сетевой интерфейс, который у меня есть.
+    # Когда будет готов интерфейс под процесс прокси, либо хардкодкожу имя интерфейса в путь к файлам, либо вынесу его в CgroupStats приватным полем
+    def get_network_stat(self, network_interface='wlp2s0'):
```

Comment:
No, let script receive the network interface name as an command line arguments

Reply:
Ok, will be done :)